README.md

```diff
@@ -71,15 +71,25 @@ Installation from source is the only currently supported format. ```deepchem```
    conda install -c omnia mdtraj
    ```
 
-7. `tensorflow`: Installing `tensorflow` on older versions of Linux (which
+7. `openmm`

```

Comment:
``` bash
conda install -y -c omnia pdbfixer=1.4
```
Instead of these two can we just install the omnia pdbfixer which is pegged to a version?